plugins/urls.py

```diff
@@ -37,9 +37,9 @@ async def github(self, msg):
         fmt = "{name} {language} {stars}: «{description}»"
         msg.reply(fmt.format(**repo))
 
-    @BotPlugin.command(github_repo + r"/(issues|pull)/(\d+)" + end_url)
+    @BotPlugin.command(github_repo + r"/(issues|pull)/(\d+)(/\w+)?" + end_url)
```

Comment:
You could also use a non-capturing group, and thus the change on the first line of this function isn't needed :wink: 
```suggestion
    @BotPlugin.command(github_repo + r"/(issues|pull)/(\d+)(?:/\w+)?" + end_url)
```

Reply:
TIL non-capturing groups